openmls/src/binary_tree/array_representation/treemath.rs

```diff
@@ -45,7 +46,7 @@ impl LeafNodeIndex {
     }
 
     /// Warning: Only use when the node index represents a leaf node
-    fn from_tree_index(node_index: u32) -> Self {
+    pub(crate) fn from_tree_index(node_index: u32) -> Self {
```

Comment:
I know that this might have been me. But can we keep these functions a little more local?